libp2p/transports/tcptransport.nim

```diff
@@ -191,6 +191,9 @@ method stop*(self: TcpTransport) {.async.} =
 
     self.servers = @[]
 
+    if self.acceptFuts.allIt(it.finished()):
```

Comment:
I interpreted @dryajov's comment as: for each future that hasn't been finished, it must be canceled.

Reply:
Yes, that should be enough. So this new check isn't necessary, the previous version without this change should be enough.

Reply:
> Yes, that should be enough. So this new check isn't necessary, the previous version without this change should be enough.

Okay perfect! I just reverted the previous commit.

@diegomrsantos - Kindly re-approve and merge if you consider it appropriate :)
Thanks!

Reply:
Sure, sorry for the trouble.

Reply:
> Sure, sorry for the trouble.

haha no worries at all! Thanks for the patience! :raised_hands: